packages/api/src/services/lsp-state.e2e.ts

```diff
@@ -0,0 +1,81 @@
+require("dotenv").config();
+import * as uma from "@uma/sdk";
+import assert from "assert";
+import { ethers } from "ethers";
+import Service from "./lsp-state";
+import type { AppState } from "../";
+import { Multicall } from "@uma/sdk";
+import { lsps } from "../tables";
+// this fixes usage of "this" as any
+import type Mocha from "mocha";
+
+type Dependencies = Pick<
+  AppState,
+  "lsps" | "registeredLsps" | "provider" | "collateralAddresses" | "shortAddresses" | "longAddresses" | "multicall"
+>;
+
+const registeredContracts = ["0x372802d8A2D69bB43872a1AABe2bd403a0FafA1F"];
+
+describe("lsp-state service", function () {
+  let appState: Dependencies;
+  before(async function () {
+    assert(process.env.CUSTOM_NODE_URL);
```

Comment:
are these tests intended to run in CI?

Reply:
nope, any test with `e2e` in the filename means it wont run in ci, its run with `yarn e2e`